tests/integration/components/donations/become-a-donor-test.js

```diff
@@ -0,0 +1,51 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import Ember from 'ember';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+
+import becomeADonorComponent from '../../../pages/components/donations/become-a-donor';
+
+const {
+  K
+} = Ember;
+
+let page = PageObject.create(becomeADonorComponent);
+
+function setHandlers(context, { becomeADonorHandler = K } = {}) {
```

Comment:
Not sure that the second param needs to be a hash here. Can we just do 
`setHandlers(context, becomeADonorHandler = K) { ... }`

Then just update line 46 to be `setHandlers(this, becomeADonorHandler)`, rather than wrapping it?